### DISCUSSION TITLE
Temporary desktop client errors for uploaded pictures on mobile client

### DISCUSSION THREAD

**Discussion by neocult-de** [nextcloud/all-in-one#5511] [Questions] [answered]
Hi there,

do anyone else see the issue, that Desktop clients (Windows and Linux Desktop client alike) receive a server error files that were uploaded via Android Clients "Instant Upload"?  I receive "Server could not complete the request" or the files is not existing anymore errors.

If I look via web interface the files are there as usual and can open and interact with them without issue. Nothing suspicious or out of the ordinary.

After around 5-45 minutes I can sync with the Desktop client them normally too and the errors disappear. 

But with the next "Instant Upload" or manual upload via Android Nextcloud Client I get the same behavior again very consistently. (Nextcloud 30 back to like Nextcloud 25).

Anyone any experience with that? Since I don´t have indications in the server log and on the website interface there is nothing for me to follow up on so far.

Looking forward to any ideas. (This error might not be AIO, related, but have it in an AIO Instance)

**Comment by szaimen** [comment, marked as answer]
Hi, please open a thread for this at https://help.nextcloud.com thanks!
